src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -10,8 +10,8 @@
     public static final String COMA = ",";
     public static final int ZERO = 0;
     public static final int ONE = 1;
-    int supply = 0;
-    int buy = 0;
+    private int supply = 0;
+    private int buy = 0;
```

Comment:
local does not mean private
these variables should be declared on the method level, not on the class level

Reply:
done